requirements.txt

```diff
@@ -1,3 +1,5 @@
 py3o.template
 py3o.formats
 genshi>=0.7
+cryptography==3.4.7
+endesive==2.0.6
```

Comment:
@Laurent-Corron please don't pin here, or use  `>=` for the version number if you know a minimum version that works.